metpetdb_api/api/samples/v1/views.py

```diff
@@ -73,13 +75,17 @@ def list(self, request, *args, **kwargs):
 
         qs = sample_qs_optimizer(params, qs)
 
-        page = self.paginate_queryset(qs)
-        if page is not None:
-            serializer = self.get_serializer(page, many=True)
-            return self.get_paginated_response(serializer.data)
+        if params.get('format') == 'csv':
+            serializer = self.get_serializer(qs, many=True)
+            return Response(serializer.data)
+        else:
+            page = self.paginate_queryset(qs)
+            if page is not None:
```

Comment:
The 'truthy' value of `None` is `False`, so you can really just write `if page` instead.

Reply:
haha, i didn't actually write this part (just scooted it under the else statement)